packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverview.tsx

```diff
@@ -236,10 +239,34 @@ export const SettingsDataModelOverview = () => {
         onEdgesChange={onEdgesChange}
         nodeTypes={NodeTypes}
         onNodesChange={handleNodesChange}
+        nodesDraggable={isInteractive}
+        elementsSelectable={isInteractive}
         proOptions={{ hideAttribution: true }}
       >
         <Background />
-        <Controls />
+
+        <IconButtonGroup
```

Comment:
Should we had a shadow on that? I don't think I see it on my side

Reply:
remainder in the general react flow css above :) should be fixed now

Reply:
Thanks!